### ISSUE TITLE
`template-indent` not check comment

### ISSUE THREAD

**Issue by tjx666** [sindresorhus/eslint-plugin-unicorn#1746]
before the tenplet string is multple line comment.

![image](https://user-images.githubusercontent.com/41773861/158319297-c216d04c-e059-4fae-879f-8a799224eebc.png)

unicorn/template-indent

```js
        const html = /* html */`<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
        <title>Scripting Listener</title>
    </head>
    <body>
        <div id="root"></div>
    </body>
</html>`;
```

**Comment by fisker** [comment]
If I remember correctly, you need add a line break after `` ` ``, can you test?

**Comment by tjx666** [comment]
![image](https://user-images.githubusercontent.com/41773861/158320613-7a818cba-1ca1-4fef-95b1-7f8141e1ae47.png)

**Comment by fisker** [comment]
So, it's reported correctly?

**Comment by fisker** [comment]
As I tested, it's working as expected.

![image](https://user-images.githubusercontent.com/172584/158321774-22eb3792-71a6-4655-95b2-e4a1fdc80897.png)

**Comment by tjx666** [comment]
I think this rule shouldn't be included in recommend,  because not many users use the library like ` strip-indent`

**Comment by tjx666** [comment]
I can use auto fix to get the code pass validate, but I hav't use any library to transform it.

**Comment by fisker** [comment]
It's all about indention, why "transform" is required?

**Comment by tjx666** [comment]
This is the result auto fixed:

![image](https://user-images.githubusercontent.com/41773861/158339902-1a3a176e-0f82-4eae-a80b-74743def739f.png)

This is original code:

![image](https://user-images.githubusercontent.com/41773861/158340003-488f16f5-fffb-440d-892b-5d087ac79319.png)

If no extra tranform, these two html code is differen't, some project may use libray to strip whitespace, but I don't.

**Comment by fisker** [comment]
Add `eslint-disable`/ remove `html` comment then
